test/Microsoft.Azure.WebJobs.Host.FunctionalTests/TestDoubles/MemoryBlobStore.cs

```diff
@@ -15,7 +15,7 @@ namespace Microsoft.Azure.WebJobs.Host.FunctionalTests.TestDoubles
     internal class MemoryBlobStore
     {
         private readonly ConcurrentDictionary<string, Container> _items = new ConcurrentDictionary<string, Container>();
-        private ServiceProperties _properties = new ServiceProperties();
+        private ServiceProperties _properties = new ServiceProperties(new LoggingProperties(), new MetricsProperties(), new MetricsProperties(), new CorsProperties());
```

Comment:
In the previous Storage package, `ServiceProperties` initialized all of its properties. Now it does not. Instead they've added a second constructor that allows you to initialize them.